About.html

```diff
@@ -0,0 +1,159 @@
+            <smal class="c-1">
+                 Brisbane 
+            </small>
+        </logo>
+
+        <!-- desk main navbar -->
+        <div class="desk-main-nav">
+          <div class="D-nav">
+            <a href="index.html" class="text-sz mx-2">Home</a>
+            <a href="#logo" class="text-sz mx-2">Logo</a>
+            <a href="#past" class="text-sz mx-2">Previous Summits</a>
+            <a href="#" class="nav-red mx-2">CC Compaign</a>
+          </div>
+        </div>
+        <ul class="mobile-menu">
+            <li class="mobile-items">
+              <a href="index.html" class="mobile-link">Home</a>
+            </li>
+            <li class="mobile-items">
+              <a href="About.html" class="mobile-link">About</a>
+            </li>
+            <li class="mobile-items">
+                <a href="#"  class="nav-red mx-2">CC Compaign</a>
+            </li>
+        </ul>   
+    </nav>
+</header>
+    <main>
+        <!-- section of summit Introduction -->
+        <section class="intro pb-5 pt-5 big-bg" id="home">
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+            <div class="d-flex flex-column">
+                <div class="mx-5">
+                    <h2 class="c-2 fs-2">"Hello! Globalized World"</h2>
+                    <h1 class="intro-title">NATIONAL EDUCATION SUMMIT
+                        <br>2023
+                    </h1>
+                </div>
+                <div class="intro-text">
+                    <p class="lead mx-3 px-2 py-4">
+                        Explore the comprehensive program on offer in Melbourne including Conferences, 
+                        Seminars and Workshops. Over the two days educators can attend Diverse Leaners Symposium, 
+                        Wellbeing for Future Focused Schools, Capacity Building School Libraries, 
+                        STEM Across the Curriculum, Literacy Masterclass, Classroom of the Future and Free Seminars.
+                    </p>
+                </div>
+                <h2 class="about-contact text-center mx-3">Please contact us per Email for any furthet questions <br>about National Eductional Summit</h2>
+                <h3 class="abou-email text-center mx-3">educationvisitor@iecgroup.com.au</h3>
+            </div>
+        </section>
+        
+        <!-- Summit logo section-->
+        <section class="border" id="logo">
+            <h2 class="logo-text text-center py-5">The National Educational Summit<br> Logo 2023</h2>
+            <div class="br-line"></div>
+            <h2 class="about-contact text-center mx-3">The logo of National Educational Summit was decided<br>through recently held logo competition</h2>
+            <div>
+                <img class="NES-logo pb-5" src="./images/NES+LOGO.png" alt="NES+LOGO">
+            </div>
```

Comment:
Great design @TiteLunda 🤝

- [ ] Kindy reduce the size of this image and make sure it is positioned the same way it is positioned in the given design. This will make your design a replica of the design provided.
- Your design
![your image](https://user-images.githubusercontent.com/63917725/235024600-bbfe6b36-4b73-4470-99fe-5f5b2b6b317d.PNG)
- Given design
![given image](https://user-images.githubusercontent.com/63917725/235024716-be5ee0f0-d2ec-4bab-bb35-269f2517c78d.PNG)